Lib/slapdtest/_slapdtest.py

```diff
@@ -187,6 +191,9 @@ class SlapdObject(object):
         'core.schema',
     )
 
+    modules = ()
+    overlays = ()
```

Comment:
Since these attributes can be overridden in subclasses, they should be documented.